Assets/Scripts/KatamariMovement.cs

```diff
@@ -134,25 +132,34 @@ private void FixedUpdate()
             heading *= Quaternion.Euler(0, yRotation, 0);
         }
 
-        float accelerateFuelUsed = resourceManager.UseFuel(hitInput);
-        float stopFuelUsed = resourceManager.UseFuel(stopInput);
-
-        rb.AddForce(CalculateHitVector(accelerateFuelUsed), forceMode);
-
-        // Roll the katamari in the direction it is being hit
-        rb.AddTorque(CalculateRollVector(accelerateFuelUsed), forceMode);
+        if (
+            (
+                movementState == StateMachine.normalSpeed ||
+                movementState == StateMachine.speedUp ||
+                movementState == StateMachine.slowMotion ||
+                movementState == StateMachine.speedUp
+            ) && !isFlyMovementDeadTime
+        )
```

Comment:
Just making sure we can't apply any fly force right after a golf hit, in line with the slow down restrictions.